help/de/Help_ch_Customize.xml

```diff
@@ -1,2481 +1,3494 @@
+              </varlistentry>
+<!-- FIXME source: new item -->
+              <varlistentry>
+                <term><guilabel>Den neuen Reiter neben dem aktuellen Reiter öffnen</guilabel></term>
 
-        <listitem>
-          <para><guilabel>Tab Position</guilabel>
-          </para>
+                <listitem>
+                  <para>Wenn aktivert, werden neue Reiter neben dem aktuellen Reiter geöffnet, ansonsten werden die neuen
+                    Reiter stattdessen am Ende geöffnet.
+                  </para>
+                </listitem>
+              </varlistentry>
 
-          <para>Position the tabs at the <guilabel>Top</guilabel> (default), <guilabel>Bottom</guilabel>,
-            <guilabel>Left</guilabel> or <guilabel>Right</guilabel>.
-          </para>
-        </listitem>
+              <varlistentry>
+                <term><guilabel>Die zuletzt benutzte Karteikarte nach vorne bringen</guilabel></term>
 
-        <listitem>
-          <para><guilabel>Summary Bar Position</guilabel>
-          </para>
+                <listitem>
+                  <para>Wenn Sie diese Option auswählen, wird beim Schließen eines Reiters der zuletzt benutzte Reiter
+                    aktiviert. Ansonsten wird der rechte Reiter angewählt.
+                  </para>
+                </listitem>
+              </varlistentry>
 
-          <para>Position the Summary Bar at the <guilabel>Top</guilabel> or at the <guilabel>Bottom</guilabel>
-            (default) of the &app; main window.
-          </para>
-        </listitem>
-      </itemizedlist>
+              <varlistentry>
+                <term><guilabel>Breite</guilabel></term>
+
+                <listitem>
+                  <para>Die Breite der Reiterbeschriftung, ausgedrückt in Zeichen.
+                    <note>
+                      <para>Wenn der Text auf dem Reiter länger als dieser Wert ist, dann wird die Reiterbeschriftung in der
+                        Textmitte so beschnitten und durch eine Ellipse ersetzt, dass diese maximale
+                        Breite erreicht wird.
+                      </para>
+                    </note>
+                  </para>
+                </listitem>
+              </varlistentry>
+            </variablelist>
+          </listitem>
+        </varlistentry>
+
+        <varlistentry>
+          <term><guilabel>Position Reiter der Karteikarten</guilabel></term>
+
+          <listitem>
+            <para>Position der Reiter <guilabel>Oben</guilabel> (Standard), <guilabel>Unten</guilabel>,
+              <guilabel>Links</guilabel> oder <guilabel>Rechts</guilabel> am Rand des Fensters.
+            </para>
+          </listitem>
+        </varlistentry>
+
+        <varlistentry>
+          <term><guilabel>Position Zusammenfassungsleiste</guilabel></term>
+
+          <listitem>
+            <para>Positioniert die Zusammenfassungsleiste <guilabel>Oben</guilabel> oder <guilabel>Unten</guilabel>
+              (Standard) am &app; Hauptfenster.
```

Comment:
(Standard) in der jeweiligen Ansicht.

Sonst wär sie ja außerhalb der Reiterleiste, wenn beide am selben Rand sind